schemas/app-interface/database-access-1.yml

```diff
@@ -0,0 +1,39 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /app-interface/database-access-1.yml
+  name:
+    type: string
+  username:
+    type: string
+  database:
+    type: string
+  delete:
+    type: boolean
+  access:
+    type: array
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        grants:
+          type: array
+          items:
+            type: string
```

Comment:
Does it make sense to have an `enum` with all possible grants here?

Reply:
I added an enum, can you please check?

Reply:
Actually, wait, I have to review the permissions again